torch/_tensor_docs.py

```diff
@@ -716,6 +716,46 @@ def add_docstr_all(method, docstr):
 See :func:`torch.diagonal`
 """)
 
+add_docstr_all('fill_diagonal_',
+               r"""
+fill_diagonal_(fill_value, wrap=False) -> Tensor
+
+Fill the main diagonal of the given tensor of any dimensionality.
```

Comment:
I'm having a bit of trouble parsing the description -- do you mean "Fill the main diagonal of a tensor that has at least 2-dimensions?"

Reply:
sorry, the description is so confused. I'm changed this.